hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
+//returns true if the given number is a palindrome, false otherwise
+const isPalindrome = (num) => {
+  const original = num.toString();
 
-elem.addEventListener('input', handleInput);
+  //reverse string by splitting it into an array and reversing that, then rejoining
+  //credit: https://www.freecodecamp.org/news/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb/
+  const reversed = original.split("").reverse().join("");
+
+  return reversed === original;
+};
+
+//Display to user whether input was a palindrome or not
```

Comment:
It's unclear in this program because I only have two lines at maximum, but I generally style my comments this way intentionally to make it clear that the later lines are a continuation of the original thought. This did lead me to noticing some of my other comments are styled inconsistently though, so I have fixed that. Thank you for the feedback.